src/main/java/hudson/plugins/jira/JiraChangeLogAnnotator.java

```diff
@@ -37,8 +37,8 @@ public JiraChangeLogAnnotator() {
     }
 
     @Override
-    public void annotate(Run<?, ?> build, Entry change, MarkupText text) {
-        JiraSite site = getSiteForProject(build.getParent());
+    public void annotate(Run<?, ?> run, Entry change, MarkupText text) {
```

Comment:
why renaming this? it's confusing as it generate some noise for nothing. (i.e make me feeling what happen here a lot of changes!)

Reply:
agree, while that changes make total sense, let's raise them as another PR

Reply:
I have to disagree here - I am visibly refactoring things *only* on the code I am touching.
I don't see a value of added work to split this into another PR, it is an extra and unneeded work.
I considered doing this in a separate commit but I decided that it's also extra and unneeded work as, again, I'm only refactoring the code I'm touching - this is opportunistic refactoring - see https://martinfowler.com/bliki/OpportunisticRefactoring.html.

Git blame and commit description provide enough description of why the change has been done (and that's why also sometimes the PRs should be squashed).